doc/generic/pgf/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+### Fixed (Alexander Grahn, 2023-09-12) 
+
+- output BBox adjustment in pgfsys-dvisvgm.def, fixes #1275
+
```

Comment:
Need to adapt to [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) style.

Reply:
I think I got the Changelog right now, thanks for providing the link.